platform/osgi/platform-osgi-configadmin/src/main/java/org/codice/felix/cm/file/CopiedEncryptionService.java

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.felix.cm.file;
+
+import java.io.File;
+
+import org.keyczar.Crypter;
+import org.keyczar.KeyczarTool;
+
+public class CopiedEncryptionService {
+    private final Crypter crypter;
+
+    private String getPasswordDirectory() {
+        final String karafHome = System.getProperty("karaf.home");
+        final String defaultPwDir =
+                "/Users/lambeaux/Connexta/instances/ddf-2.11.0-SNAPSHOT/etc/certs";
```

Comment:
Residual code from troubleshooting. Flagged for removal. While this whole method is technically not necessary, I would then pose the question about how we should fail if the `karaf.home` system property comes back null. I'm not sure at what bundle level this could happen, if at all, so it may be a moot point. In our case, it has never come back null.